05-prework-exercises/02-restaurant-bill/src/restaurantBill.js

```diff
@@ -0,0 +1,9 @@
+const restaurantBill = (cuenta) => {
```

Comment:
que les parece si la cantidad de amigas tb es un parametro
```suggestion
const restaurantBill = (cuenta, cantDeAmigas) => {
```

Reply:
hecho